src/widget/form/addfriendform.cpp

```diff
@@ -228,7 +228,7 @@ void AddFriendForm::setIdFromClipboard()
     }
 }
 
-void AddFriendForm::deleteFriendRequest(const QString& toxId)
+void AddFriendForm::deleteFriendRequest(const QString& toxId) const
```

Comment:
Actually code should be moved to `Settings` and this should change to:

 ```c++
static inline void AddFriendForm::deleteFriendRequest(const QString& toxId)
{
    Settings::getInstance().removeFriendRequest(toxId);
}
```

As the `AddFriendForm` will only be a single instance (when adding a friend), it actually makes no difference, if it is static or not.